src/main/java/jp/ne/paypay/model/Payment.java

```diff
@@ -289,7 +296,31 @@ public String getAmountDescription() {
   public void setAmountDescription(String amountDescription) {
     this.amountDescription = amountDescription;
   }
-  
+
+
+  public String getPaymentMethodType() {
+    return paymentMethodType;
+  }
+
+  public String getPaymentMethodId() {
+    return paymentMethodId;
+  }
+
+  public void setPaymentMethod(PaymentMethod paymentMethod) {
+    this.paymentMethodType = paymentMethod.getPaymentMethodType();
+    this.paymentMethodId = paymentMethod.getPaymentMethodId();
+  }
+
+  /**
+   * Set WALLET as payment method.
+   * As of now, WALLET is the default payment method, and
+   * it is not required to send paymentMethodId if "WALLET" is passed as paymentMethodType.
+   */
+  public void setPaymentMethodAsWallet() {
+    this.paymentMethodType = "WALLET";
+    this.paymentMethodId = null;
+  }
```

Comment:
[design explanation]

In the API doc, it is implied that users can provide `WALLET` as well as `PAY_LATER_CC` and `CREDIT_CARD`.

> If "PAY_LATER_CC", "WALLET" or "CREDIT_CARD" is passed, a payment transaction is expected to happen with PaylaterCC, PayPay Balance or Credit Card as payment method ... [reference](https://www.paypay.ne.jp/opa/doc/v1.0/direct_debit.html#tag/Payment/operation/createPayment)

However, `getPaymentMethods` currently does not return `WALLET`.

> This API returns the available payment method type and payment method id of a user, which could be "PAY_LATER_CC" (learn more about PAY_LATER_CC) or "CREDIT_CARD". This API does not return the user's wallet payment method id, because it is the default payment-method when no paymentMethodId is passed to the payment API.  [reference](https://www.paypay.ne.jp/opa/doc/v1.0/direct_debit.html#tag/Payment/operation/GetPaymentMethods)

Although there's no need to explicitly set WALLET for payments as wallet is used as default, providing a way to set "WALLET" aligns to the API doc.

Reply:
Removed because setter implementation has changed and no need for this.